packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -82,7 +81,7 @@ class PaginatedDataTable extends StatefulWidget {
     this.onRowsPerPageChanged,
     this.dragStartBehavior = DragStartBehavior.start,
     required this.source,
-  }) : assert(header != null),
+  }) : assert(actions == null || (actions != null && header != null)),
```

Comment:
Ah, this is just so that if actions is set, then header must be set.

Basically:

| Header  | Actions | Result |
| ------------- | ------------- | ------------- |
| Null  | Null  | Good |
| Widget  | Null  | Good |
| Null  | Widget  | Assertion error |
| Widget  | Widget | Good |

Reply:
Ohhh, thanks